### PR TITLE
New serverless pattern - lambda-ecr

### PR DESCRIPTION
**Pull request by nj80750**
*Issue #, if available:* https://github.com/aws-samples/serverless-patterns/issues/1950

*Description of changes:*
This pattern will demonstrate integration of Lambda and ECR

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: terraform-lambda-ecr-integration/templates/lambda-role.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "BasicDescribePolicy",
+      "Effect": "Allow",
+      "Action": [
+        "cloudwatch:DescribeAlarms",
+        "cloudwatch:PutMetricAlarm",
```
Comment (pputhran) [inline terraform-lambda-ecr-integration/templates/lambda-role.json:9]: Can you please expand the need for DescribeAlarms and PutMetricAlarm permissions here?

**Review by pputhran — COMMENTED** [review]
Question reg CloudWatch alarm permissions.

File: terraform-lambda-ecr-integration/templates/lambda-role.json (hunk 1 above)
Comment (nj80750) [reply terraform-lambda-ecr-integration/templates/lambda-role.json:9]: I added it to give flexibility to add alarms in the cloud watch but for this particular use case it is not needed. I will remove it and push the code again.

**Review by pputhran — APPROVED** [review]
Approved

**Pull request merged**